store/tikv/lock_resolver.go

```diff
@@ -576,7 +590,6 @@ func (lr *LockResolver) getTxnStatus(bo *Backoffer, txnID uint64, primary []byte
 			}
 
 			status.commitTS = cmdResp.CommitVersion
-			lr.saveResolved(txnID, status)
```

Comment:
`getTxnStatus` seems has some other caller gc's `BatchResolveLocks`  and  binlog's [GetTxnStatus](https://github.com/pingcap/tidb-binlog/blob/27030ddae9ec0e42f371694435245e9533e64dec/pump/storage/storage.go#L496) , does this will have bad effact on them if we move this line out of the method?

PS: maybe we need upgrade binlog & [cdc](https://github.com/pingcap/ticdc/blob/master/cdc/kv/client.go#L1352)'s  tidb dependence version after this PR merge?

Reply:
Should cache the result in `BatchResolveLocks`. I think the effect on `GetTxnStatus` is ignorable.

Reply:
The logic is added.